### PR TITLE
Update inputevent.rst: set_process() changes for Godot 3

### PR DESCRIPTION
**Pull request by mhilbrunner**

### CODE REVIEW

File: learning/features/inputs/inputevent.rst [hunk 1]
```diff
@@ -32,7 +32,8 @@ received input, in order:
 
 1. First of all, the standard _input function
    will be called in any node with input processing enabled (enable with
-   :ref:`Node.set_process_input() <class_Node_set_process_input>` and override
+   :ref:`Node.set_process_input() <class_Node_set_process_input>` or
+   simply override :ref:`Node._process() <class_Node__process>` which enables it automatically, and override
```
Comment (akien-mga) [inline learning/features/inputs/inputevent.rst]: That's incorrect, `_process` and `_input` are unrelated. If you have `_input`, input processing is enabled automatically in 3.0, contrarily to 2.1. Therefore `set_process_input` is only useful to disable the auto-enabled processing, or reenable it later.

Comment (mhilbrunner) [reply learning/features/inputs/inputevent.rst]: Ah, of course you're right :)

**Comment by mhilbrunner** [comment]
Fixed, this should be correct now :)

**Comment by mhilbrunner** [comment]
Anything I should update on this? @cbscribe?

**Comment by cbscribe** [comment]
Looks good to me.

**Pull request merged**